content/docs/internals/special-routes.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Special Routes
+lang: en-US
+keywords:
+  [
+    pomerium,
+    identity access proxy,
+    special routes,
+    authentication endpoints,
+    api,
+    user information,
+    session management,
+    device identity,
+    webauthn,
+    programmatic access,
+    logout flow,
+    jwt,
+  ]
+description: Learn about Pomerium's reserved endpoints that handle authentication flows, session management, and user information retrieval.
+sidebar_position: 5
+---
+
+# Special Routes
+
+Pomerium's proxy service reserves the `/.pomerium` path for internal endpoints that facilitate authentication flows, session management, and user information retrieval. Any request to a route ending in `/.pomerium/...` is intercepted by Pomerium (not forwarded to upstream apps) and handled by these special endpoints. This page documents all such endpoints, their purpose, and usage for end-users and developers integrating with Pomerium.
+
+Below is a summary of the special `/.pomerium` routes, including their HTTP methods, purpose, and whether an existing authenticated session is required:
+
+| **Endpoint** | **Method(s)** | **Requires Auth?** | **Purpose** |
+| --- | --- | --- | --- |
+| [`/.pomerium/`](#user-info-page) | `GET` | Yes (must be logged in) | User info page (HTML) showing current identity, session details, and device identities. |
```

Comment:
This pages actually doesn't require login. The page shows a message like:

> User Details Not Available
> Have you signed in yet?

Reply:
Does it return a `403` or `200` status code?

Reply:
It returns a 200